gui/WorldView.cxx

```diff
@@ -441,6 +473,126 @@ void WorldView::setBackgroundColor(QColor const& color)
 }
 
 //-----------------------------------------------------------------------------
+void WorldView::setActiveDepthMap(vtkMaptkCamera* camera, QString vtiPath)
+{
+  QTE_D();
+
+  if(d->UI.actionDepthmapDisplay->isChecked() && !d->depthmapAlreadLoaded)
+  {
+    std::string filename = vtiPath.toStdString();
+
+    vtkNew<vtkXMLImageDataReader> readerIm;
+
+    readerIm->SetFileName(filename.c_str());
+    readerIm->Update();
+
+    int width = readerIm->GetOutput()->GetDimensions()[0];
+    int height = readerIm->GetOutput()->GetDimensions()[1];
+
+    vtkNew<vtkPoints> points;
+
+    points->SetNumberOfPoints(width*height);
```

Comment:
Should this be `SetNumberOfPoints(readerIm->GetOutput()->GetNumberOfPoints())`?

Maybe you expect them to be the same for "reasonable data", but is that enforced by `vtkXMLImageDataReader`?

Also, there is at least one way I've crashed this method due to bad data.